include/library/spdm_lib_config.h

```diff
@@ -323,7 +323,8 @@
  *
  * If chunking is not supported, it may be just LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE.
  * If chunking is supported, it should be at least LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE
- * + LIBSPDM_MAX_SPDM_MSG_SIZE + LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE.
+ * + LIBSPDM_MAX_SPDM_MSG_SIZE + LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE. 0 ~ LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE
+ * is for Secure message enconding/decoding. LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE ~ end is for Chunking.
  *
```

Comment:
Please add below table to make 3 sections clear:
```
 * +----------------------------------+-------------------------+----------------------------------+
 * |LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE|LIBSPDM_MAX_SPDM_MSG_SIZE|LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE|
 * +----------------------------------+-------------------------+----------------------------------+
 * |<-Secure msg enconding /decoding->|<-Full SPDM msg (chunk)->|<-        ???                   ->|

```

Reply:
@richkong88 , could you please comment the usage of three sections in scratch buffer in chunk mode?